src/main/java/ch/bfh/bti7081/model/manager/SeminarCategoryManager.java

```diff
@@ -16,6 +16,7 @@
     SeminarCategoryRepository seminarCategoryRepository;
 
     public List<SeminarCategory> getSeminarCategories() {
+        // we get here all categories, since we see no point in limiting the selection on the UI
```

Comment:
Naja, der Grund könnte sein, dass das UI mit vielen Daten unübersichtlich wäre.
Würde vorschlagen: we get all categories, as currently, we do not have the possibility for users to add categories and therefore have strict control over the present categories.

Reply:
Nee, ich hatte gerade letztens in meinem Arbeits-Projekt so einen Fall... Bei einem Dropdown macht es keinen Sinn und macht nur Probleme wenn du das einschränkst (wenn es keine Kriterien wie z.B Rechte oder ähnliches gibt)

Reply:
Jep, das könnte man in den Kommentar schreiben. 😄 
We SEE no point, ist eine Einladung uns Punkte abzuziehen da wir wieder einmal zu dumm sind, und ist auch nicht die abschliessende Begründung.